test/built-ins/Array/from/argument-passed-null.js

```diff
@@ -0,0 +1,13 @@
+// Copyright 2015 Microsoft Corporation. All rights reserved.
+// This code is governed by the license found in the LICENSE file.
+
+/*---
+description: this argument is passed as null
+es6id: 22.1.2.1
+---*/
+
+function mapFn(value, index) {
+    assert("mapFn is not expected to be called. Called for " + value + " and index " + index);

```

Comment:
The idiomatic way to express an unreachable code path is to use [the `$ERROR` functon](https://github.com/tc39/test262/blob/d62f3a82b97dba22a0be2798fbe34548e96a439f/CONTRIBUTING.md#test-environment). This will make the test source and the produced error easier to understand.